src/catalogue/items/catalogueItemsDialog.component.tsx

```diff
@@ -386,8 +386,18 @@ function CatalogueItemsDialog(props: CatalogueItemsDialogProps) {
 
               if (response && error.response?.status === 409) {
                 if (response.detail.includes('child elements')) {
+                  // find the name of the manufacturer, so it can be used in the error message
+                  const manufacturerName = manufacturerList?.find(
+                    (manufacturer) => manufacturer.id === selectedCatalogueItem.manufacturer_id
+                  ) || null
+                  // add the manufacturer name into the error message
+                  const message = response.detail.replace(
+                    "so the following fields cannot be updated: manufacturer_id, properties",
+                    ("so you cannot update the properties, and the manufacturer cannot be changed from "
+                      + manufacturerName?.name)
+                  )
```

Comment:
Instead of replacing the text directly in the backend response, consider moving this string transformation entirely to the frontend. This approach will ensure consistency in the response formatting, prevent future inconsistencies, and make it easier to manage localization or text changes.